presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/groupcreation/name/navigation/GroupCreationNameNavigation.kt

```diff
@@ -0,0 +1,18 @@
+package com.mashup.gabbangzip.sharedalbum.presentation.ui.groupcreation.name.navigation
+
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import com.mashup.gabbangzip.sharedalbum.presentation.ui.groupcreation.name.GroupCreationNameScreen
+import com.mashup.gabbangzip.sharedalbum.presentation.ui.groupcreation.navigation.GroupCreationRoute
+
+fun NavGraphBuilder.groupCreationNameNavGraph(
+    onBackButtonClicked: () -> Unit,
+    onNextButtonClicked: () -> Unit,
+) {
+    composable(route = GroupCreationRoute.SecondScreenRoute.route) {
```

Comment:
```suggestion
    composable(route = GroupCreationRoute.NameScreenRoute.route) {
```
여기 Route 이름도 새로 NameRoute 로 추가해주어야할 것 같은데....
보니까 나도 안바꿨었네..


```kotlin
sealed interface GroupCreationRoute {
    val route: String

    data object IntroScreenRoute : GroupCreationRoute {
        override val route: String = "groupIntroScreenRoute"
    }

    data object NameScreenRoute : GroupCreationRoute {
        override val route: String = "groupNameScreenRoute"
    }

    companion object {
        val initRoute = IntroScreenRoute.route
    }
}
```
혹시 Route 이름 변경하는김에 Intro 도 같이 바꿔줄 수 있을까... ㅎㅎ..(머쓱)

Reply:
그렇네 나도 안 바꿨구나 헿 acb5a13b5d06d7366d8c4235a220c6cd3096a3ef